docs/win-install.md

```diff
@@ -1,40 +0,0 @@
-# Windows installation problems
```

Comment:
I think everything in this document was to do with native installation tools, so it is no longer needed.